src/test/java/software/amazon/smithy/lsp/document/DocumentParserTest.java

```diff
@@ -319,4 +328,58 @@ enum Baz {
         assertThat(getInputA.kind(), equalTo(DocumentShape.Kind.DefinedMember));
         assertThat(getInputA.shapeName(), string("a"));
     }
+
+    @ParameterizedTest
+    @MethodSource("contiguousRangeTestCases")
+    public void findsContiguousRange(SourceLocation input, Range expected)  {
```

Comment:
Can you add one where the source location is within whitespace? Not sure if that will ever be the case, but just in case.

Reply:
Sure. Can add that.